native/wasmex/todo-list.wit

```diff
@@ -0,0 +1,6 @@
+package local:todo-list;
+
+world todo-list {
+  export init: func() -> list<string>;
+  export add-todo: func(todo: string, todos: list<string>) -> list<string>;
+}
```

Comment:
is this todo-list still in use? (same for the other todo-list.wit file and the .wasm file)

Reply:
nope just stuff left around when I was in "experiment" mode